pandas/tests/arrays/test_period.py

```diff
@@ -1,3 +1,5 @@
+from distutils.version import LooseVersion
```

Comment:
can compat._optional.import_optional_dependency be used to simplify things?

Reply:
From quickly looking at that function, I *think* it can only be used for lazy imports, which is not what I am doing here (but that is exactly something that we might need to discuss)